### PR TITLE
Merge CASP branch to master

### PR DESCRIPTION
**Pull request by lslezak**
This is just a `CASP` -> `master` merge, it has been already reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ YaST Installation Framework
 
 [![Travis Build](https://travis-ci.org/yast/yast-installation.svg?branch=master)](https://travis-ci.org/yast/yast-installation)
 [![Jenkins Build](http://img.shields.io/jenkins/s/https/ci.opensuse.org/yast-installation-master.svg)](https://ci.opensuse.org/view/Yast/job/yast-installation-master/)
-
+[![Coverage Status](https://coveralls.io/repos/github/yast/yast-installation/badge.svg?branch=SLE-12-SP2-Octopus)](https://coveralls.io/github/yast/yast-installation?branch=SLE-12-SP2-Octopus)
```
Comment (jreidinger) [inline README.md]: this looks very wrong coverage for branch Octopus, that do not even exists

File: package/yast2-installation.changes [hunk 2]
```diff
@@ -1,4 +1,15 @@
 -------------------------------------------------------------------
+Mon Oct 31 13:23:38 UTC 2016 - lslezak@suse.cz
+
+- Added support for read-only proposal modules (FATE#321739)
+- 3.2.6
+
+-------------------------------------------------------------------
+Thu Oct 27 15:09:58 CEST 2016 - shundhammer@suse.de
+
+- Documentation for subvolumes in control.xml (fate#321737)
```
Comment (jreidinger) [inline package/yast2-installation.changes]: inconsistent acronym case. It should be `fate#` according to https://en.opensuse.org/openSUSE:Packaging_Patches_guidelines#Current_set_of_abbreviations

File: src/lib/installation/proposal_store.rb [hunk 3]
```diff
@@ -237,16 +238,39 @@ def id_for(client)
...
+      title = description["rich_text_title"] ||
         description["rich_text_raw_title"] ||
         client
+
+      return title unless read_only?(client)
+
+      # remove any HTML links if the proposal is read only,
+      # use the non-greedy .*? repetition to handle
+      # the "<a>foo</a> <a>bar</a>" case correctly
+      title.gsub(/<a.*?>(.*?)<\/a>/, "\\1")
```
Comment (jreidinger) [inline src/lib/installation/proposal_store.rb]: thinking about this, I think it can be achieved without non greedy regexp via `/<a[^>]*>([^<]*)<\/a>/`

Comment (jreidinger) [inline src/lib/installation/proposal_store.rb]: but just NP

File: README.md (hunk 1 above)
Comment (lslezak) [inline README.md]: Ouch, I overlooked this...

File: src/lib/installation/proposal_store.rb (hunk 3 above)
Comment (lslezak) [inline src/lib/installation/proposal_store.rb]: Um, compared with the non-greedy solution this does not work well with nested tags:
```
irb(main):001:0> "<a href='foo'><b>Foo</b></a>".gsub(/<a.*?>(.*?)<\/a>/, "\\1")
=> "<b>Foo</b>"
irb(main):002:0> "<a href='foo'><b>Foo</b></a>".gsub(/<a[^>]*>([^<]*)<\/a>/, "\\1")
=> "<a href='foo'><b>Foo</b></a>"
```

So I'd keep the original version...

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**